product-service/swagger.yml

```diff
@@ -0,0 +1,88 @@
+swagger: '2.0'
+info:
+  description: NodeJS-AWS-BE API
+  version: 1.0.0
+  title: NodeJS-AWS-BE API
+  # put the contact info for your development or API team
+  contact:
+    email: credo166@mail.ru
+
```

Comment:
I would highly appreciate servers section in the API so it can be tested right in swagger.

```
servers:
  - url: 'https://<hash>.execute-api.eu-west-1.amazonaws.com/dev'
```